src/test/java/dk/cngroup/lentils/repository/HintTakenRepositoryTest.java

```diff
@@ -55,6 +61,17 @@ public void saveAllHintsTakenForOneCypherOneTeamTest() {
         assertEquals(ObjectGenerator.NUMBER_OF_HINTS_FOR_CYPHER, hintTakenRepository.count());
     }
 
+    @Test
+    public void countNumberOfHintsTakenByTeamTest(){
```

Comment:
Je supr, ze jsi na to udelal test, ale v tom pripade, by si ho mohl udelat poradne a overit vic situaci ;) :D (0, 1, vice, odlisny tym, odlisna sifra, ...).

Reply:
ono to původně bylo pro metodu, která pak nebyla potřebná, takže je smazaná, ale nějaké ty testy rád přidám :D